### PR TITLE
Blank Page Fix

### PR DESCRIPTION
**Pull request by drewpate**
<Update pull_request_template.md to change this template>
<Delete any sections below that are not applicable>

# Description

Issue: #37 >

- [ x ] Change 1 fix base in vite.congfig.ts

### CODE REVIEW

**Review by codeTrackLift — APPROVED** [review]
Thanks for updating this @drewpate.  Approved, just a minor nit/question: 
- Is `next-env.d.ts` needed? I _think_ this is only needed for a `next` project?

**Comment by drewpate** [comment]
> Thanks for updating this @drewpate.  Approved, just a minor nit/question: 
> 
> - Is `next-env.d.ts` needed? I _think_ this is only needed for a `next` project?

In that case I don't believe it is and we could safely remove it.

File: next-env.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
```
Comment (HudsonGraeme) [inline next-env.d.ts:1]: This file can be omitted since it was a typescript declaration file for NextJS however we no longer use next :)

**Review by HudsonGraeme — COMMENTED** [review]
LGTM! Just one extra file here that can be removed :)

**Comment by drewpate** [comment]
@HudsonGraeme @codeTrackLift I had hopes this would fix the blank page but it looks like something still isn't quite right. The URL is definitely the issue though.

**Comment by HudsonGraeme** [comment]
Ahhhh I understand now what the issue is, have run into this in the past but forgot the learned knowledge 😬 
For local dev we need `/` and for production we need to use `/learnjs`

**Pull request merged**